controllers/tests/config/selfnoderemediationconfig_controller_test.go

```diff
@@ -39,18 +44,40 @@ var _ = Describe("SNR Config Test", func() {
 
 	})
 
-	Context("DS installation", func() {
-		key := types.NamespacedName{
-			Namespace: shared.Namespace,
-			Name:      dsName,
+	AfterEach(func() {
+		if isSkipCleanup {
+			isSkipCleanup = false
+			return
 		}
 
+		tmpConfig := &selfnoderemediationv1alpha1.SelfNodeRemediationConfig{}
+		tmpDs := &appsv1.DaemonSet{}
+		//verify ds and config exist
+		Eventually(func(g Gomega) bool {
+
+			g.Expect(k8sClient.Get(context.Background(), client.ObjectKeyFromObject(config), tmpConfig)).To(Succeed())
+			g.Expect(k8sClient.Get(context.Background(), dsKey, tmpDs)).To(Succeed())
+			return true
```

Comment:
you never return false, so the bool is not needed. You can Just check for Succeed() below